lib/synapse_pay_rest/api/atms.rb

```diff
@@ -0,0 +1,35 @@
+module SynapsePayRest
+  # Wrapper class for /client endpoint
+  class Atms
+    VALID_QUERY_PARAMS = [:page, :per_page, :radius, :zip, :lat, :lon].freeze
+
+    # @!attribute [rw] client
+    #   @return [SynapsePayRest::HTTPClient]
+    attr_accessor :client
+
+    # @param client [SynapsePayRest::HTTPClient]
+    def initialize(client)
+      @client = client
+    end
+    
+    # Sends a GET request to /nodes endpoint to locate atms, and returns the
```

Comment:
I think I totally misunderstand Synapse's business if `/atms` is at the top of their priority list. 😄